engineering-toolkit/eslint-config/.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": false,
+  "printWidth": 120
+}
```

Comment:
Question(optional): Shouldn't eslint-config use prettier-config and vice versa? 😄